packages/react-theme-provider/src/components/ThemeProvider/renderThemeProvider.tsx

```diff
@@ -9,7 +9,7 @@ export function renderThemeProvider(state: ThemeProviderState) {
   return (
     <slots.root {...slotProps.root}>
       <ThemeContext.Provider value={state.theme}>
-        <ThemeClassNameContext.Provider value={state.themeClassName}>{state.children}</ThemeClassNameContext.Provider>
```

Comment:
Should we remove `themeClassname` from `state`?

Reply:
yes, it's a good call out. I'll remove it